bot/kodiak/app_config.py

```diff
@@ -55,11 +54,17 @@ def __call__(
         default=["pull_request", "pull_request_review", "pull_request_comment"],
     )
 )
-USAGE_REPORTING_POOL_SIZE = config("USAGE_REPORTING_POOL_SIZE", cast=int, default=50)
 USAGE_REPORTING_QUEUE_LENGTH = config(
     "USAGE_REPORTING_QUEUE_LENGTH", cast=int, default=10_000
 )
 INGEST_QUEUE_LENGTH = config("INGEST_QUEUE_LENGTH", cast=int, default=1_000)
+BLOCKING_POP_TIMEOUT_SEC = config("BLOCKING_POP_TIMEOUT_SEC", cast=int, default=10)
```

Comment:
thoughts on calling this `REDIS_BLOCKING_POP_TIMEOUT_SEC`?
```suggestion
REDIS_BLOCKING_POP_TIMEOUT_SEC = config("REDIS_BLOCKING_POP_TIMEOUT_SEC", cast=int, default=10)
```